MonoGame.Framework.Content.Pipeline/Serialization/Compiler/ContentWriter.cs

```diff
@@ -172,7 +172,14 @@ public override void Flush()
                     if (compressedStream != null)
                         compressedStream.Dispose();
                 }
-            }
+            }
+        }
+
+        /// <summary>
+        /// Clears buffer using BinaryWriter's flush method
+        /// </summary>
+        public override void Flush()

```

Comment:
Do we need this anymore? If it's defaulting to the `virtual base` call anyways.

Reply:
If all it's doing is calling the base, then it's not needed

Reply:
I can certainly go and delete it if it's only calling base and has no other purpose

Reply:
Deleted! Apologies for the delay.

Reply:
I am not a maintainer so you would need to ping @harry-cpp / @mrhelmut to review the PR(s).